### ISSUE TITLE
Installation Fail On Amazon Linux?

### ISSUE THREAD

**Issue by slimandslam** [ansible-semaphore/semaphore#81]
Running Amazon Linux (CentOS 6.x).  1 gig of RAM. t2.micro instance.
Redis 3.0.7
MySQL 5.5.46
Python 2.7.10

```
./semaphore_linux_amd64 -setup

Hello, you will now be guided through a setup to:

- Set up configuration for a MySQL/MariaDB database
- Set up redis for session storage
- Set up a path for your playbooks
- Run DB Migrations
- Set up your user and password

DB Hostname (default 127.0.0.1:3306):
DB User (default root):
DB Password: xxxx
DB Name (default semaphore):
Redis Connection (default 127.0.0.1:6379):
Playbook path (will be auto-created if does not exist):
Config:
{
	"mysql": {
		"host": "127.0.0.1:3306",
		"user": "root",
		"pass": "xxxx",
		"name": "semaphore"
	},
	"session_db": "127.0.0.1:6379",
	"mandrill": {
		"username": "",
		"password": ""
	},
	"port": "",
	"bugsnag_key": "",
	"tmp_path": "/tmp/semaphore"
}

Is this correct? (yes/no): yes
Running: mkdir -p /tmp/semaphore
Configuration written to /tmp/semaphore

Pinging database...
Pinging redis...

Running DB Migrations
Creating migrations table
Executing migration v0.0.0 (at 2016-04-27 15:55:03.261354412 +0000 UTC)...
 [1/11]
 ERR! Query: create table user (
	`id` int(11) not null auto_increment primary key,
	`created` datetime not null default NOW(),
	`username` varchar(255) not null comment "Username, unique",
	`name` varchar(255) not null comment "Full name",
	`email` varchar(255) not null comment "Email, unique",
	`password` varchar(255) not null comment "Password",

	unique key `username` (`username`),
	unique key `email` (`email`)
) ENGINE=InnoDB CHARSET=utf8

Rolling back v0.0.0 (time: 2016-04-27 15:55:03.261781902 +0000 UTC)...
Rollback SQL doesn't exist.

[ec2-user@ip-10-0-0-161 ~]$ panic: Error 1067: Invalid default value for 'created'

goroutine 1 [running]:
panic(0xa88e60, 0xc8200f3c20)
	/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
main.doSetup(0x0)
	/Users/m/p/semaphore/main.go:136 +0xd64
main.main()
	/Users/m/p/semaphore/main.go:27 +0x30
2016/04/27 15:55:03 notifying bugsnag: Error 1067: Invalid default value for 'created'
2016/04/27 15:55:03 bugsnag.Notify: not notifying in development
```

**Comment by matejkramny** [comment]
I suppose your mysql is in strict mode, there are ways to turn it off..

I will fix the sql things though

**Comment by slimandslam** [comment]
I just yum installed mysql-server and then ran the "mysql_secure_installation" script. That's it. Pretty typical.

**Comment by matejkramny** [comment]
Mind trying again with the latest release?

**Comment by slimandslam** [comment]
It's on my list.

J

**Comment by matejkramny** [comment]
Fixed with 977835f92354a37786b582114ff525ea4e5c22b4
